Source/Resources/Content/MSGraph/DeviceManagement/DeviceEnrollmentConfigurations/deviceEnrollmentPlatformRestrictionsConfiguration - All users and all devices.json

```diff
@@ -21,7 +21,7 @@
       }
     }
   ],
-  "description": "This is the default Device Type Restriction applied with the lowest priority to all users regardless of group membership.",
+  "description": "What is it: The default device type restriction applied with the lowest priority to all users regardless of group membership. \n Simeon Notes: Simeon's baseline allows all corporate devices to be enrolled, but blocks all personally owned devices. \n Learn More: https://docs.microsoft.com/en-us/mem/intune/enrollment/enrollment-restrictions-set#create-a-device-type-restriction",
```

Comment:
**What does this do?**
Restricts the type of device that can be enrolled. The baseline allows enrollment of all corporate devices and blocks all personal devices. This limit can be overridden by other configurations with a higher priority.

**Why should you use this?**
If you want to ensure that only certain types of devices are enrolled.

**What is the end user impact?**
Users will be restricted to enrolling corporate devices only.

**Learn more**
[Create a device type restriction](https://docs.microsoft.com/en-us/mem/intune/enrollment/enrollment-restrictions-set#create-a-device-type-restriction)